packages/evm/src/verkleAccessWitness.ts

```diff
@@ -276,11 +259,11 @@ export class AccessWitness implements AccessWitnessInterface {
   }
 
   /**Create a shallow copy, could clone some caches in future for optimizations */
-  shallowCopy(): AccessWitness {
-    return new AccessWitness({ verkleCrypto: this.verkleCrypto })
+  shallowCopy(): VerkleAccessWitness {
```

Comment:
Do we actually need this? I don't see it referenced anywhere in the codebase?  Doesn't seem like it does anything other than just instantiate a brand new access witness?  Also, the code doc references caches and stuff and the `AccessWitness` class doesn't appear to have any caches.

Reply:
Hmm good point, perhaps something that was at some point used but then left out, will remove

Reply:
Ah, actually we are using it here: https://github.com/ethereumjs/ethereumjs-monorepo/blob/evm/access-witness-refactor/packages/vm/src/runTx.ts#L200

so I won't remove

Reply:
I think my point here is that `shallowCopy` here isn't copying anything. It's just calling the access witness constructor again. Couldn't we just replace the call in`runTx` with a call to the constructor and then remove this code?

Reply:
Ah yeah that's right. Removed